### PR TITLE
Klarocca/grid layout

### PR DESCRIPTION
**Pull request by kimlarocca**

### CODE REVIEW

File: app/styles/themes/gothamist/_theme.variables.scss [hunk 1]
```diff
@@ -96,11 +96,11 @@
   --max-width:      1024px;
   --max-width-xl:   1130px;
   --heading-height: 72px;
-  
+
   // Grid
-  --col-width: 48.5px;
```
Comment (walsh9) [inline app/styles/themes/gothamist/_theme.variables.scss:101]: I think this is still used in `_layout.wrappers.scss`. Is there going to be a follow up to this PR to work on the layout containers there?

File: app/styles/themes/gothamist/_theme.variables.scss [hunk 2]
```diff
@@ -96,11 +96,11 @@
...
-  --gutter:    18px;
-  
+  --gutter:       18px;
+  --gutter-large: 40px;
```
Comment (walsh9) [inline app/styles/themes/gothamist/_theme.variables.scss]: Can we rename these to `--grid-gutter` and `--grid-gutter-large`?

File: stories/10-For Developers/SampleCode/Layout.stories.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
+import {compile} from 'ember-source/dist/ember-template-compiler';
+
+export default {title: 'For Developers/Sample Code/Layouts'};
```
Comment (walsh9) [inline stories/10-For Developers/SampleCode/Layout.stories.js:3]: We've been leaning towards including our stories in `.mdx` files. This gives us more control over the layout of the docs pages. It's a little more verbose but I'd like to keep this consistent across the docs. There are a handful of ways to include stories with storybook and I'd prefer if we only use one of them.

There's an example here and we can go over this in more detail together if you want.
https://nypr-design-system.nypr.digital/latest/index.html?path=/docs/for-developers-documenting-components--page#creating-a-story-file

**Comment by kimlarocca** [comment]
@walsh9 I updated everything per your feedback, including the layouts file with the updated variable names. I still think we should move away from any kind of fixed numbers in our grid, but we can always figure that out later.

**Pull request merged**